bodhi/server/config.py

```diff
@@ -443,6 +443,9 @@ class BodhiConfig(dict):
         'waiverdb.access_token': {
             'value': None,
             'validator': _validate_none_or(six.text_type)},
+        'koji_build_web_url': {
+            'value': 'https://koji.fedoraproject.org/koji/search',
```

Comment:
Let's make this default to ```https://koji.fedoraproject.org```. That way we can use it for other Koji things if needed later, and also the name doesn't imply that it's just a URL to searching.